library/systemd/src/lib/yast2/service_widget.rb

```diff
@@ -80,86 +102,90 @@ def handle_input(event_id)
     # Stores current configuration. Should be called always even when going
```

Comment:
NOTE: This comment is for method `handle_input`. The documentation of `handle_input` says `@return [void]`. Is that correct?

Reply:
in fact no. It is comment for store. You should call store after leaving dialog to remember user selection, so when going again forward it will be set correctly. Maybe I wrote it badly?

handle_input should has nil as return as it means in cwm to continue with dialog and not return given value from it.